meteor/scripts/common.py

```diff
@@ -34,6 +34,11 @@
 PHASE_COLUMN_NAME: str = "PHI"
 DEFAULT_OUTPUT_MTZ: Path = Path("meteor_difference_map.mtz")
 DEFAULT_OUTPUT_METADATA_FILE: Path = Path("meteor_metadata.json")
+NEGATIVE_NEGENTROPY_WARNING_MESSAGE: str = """\
```

Comment:
Edits:
Negentropy estimate is negative, but by definition negentropy is a positive value. This indicates the map is very close to Gaussian and the true negentropy is close to zero. Your map may have too little non-Gaussian signal for `meteor` to optimize.